public/index.html

```diff
@@ -5,7 +5,7 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>MIME</title>
+  <title>Interspect</title>
```

Comment:
Is this official now ? Are we calling this Interspect?

Reply:
No, sorry to be unclear—just changed for the demo after our discussion on Thursday. Happy to continue talking about what we want to call it.